dirs/dirs.go

```diff
@@ -83,6 +84,7 @@ func SetRootDir(rootdir string) {
 
 	SnapSnapsDir = filepath.Join(rootdir, "/snap")
 	SnapDataDir = filepath.Join(rootdir, "/var/snap")
+	SnapDataHomeDir = "snap"
```

Comment:
Sure, I  was wondering what do do about it myself, since it is referenced only from one place I will treat it just as `"common"` i.e. -- hardcode it  in the method.